docs/Deploying-Pega-on-GKE.md

```diff
@@ -523,6 +523,18 @@ tier:
       #  e.g. web.mypega.example.com
       domain: "**gke.web.dev.pega.io**"
 ```
+18. To apply additional configuration for GKE Load balancer, configure BackendConfig with a referenced service port and associate necessary parameters as per need.
```

Comment:
Probably describe the overridable elements like `drainingTimeoutSec` or `timeOutSec` that are mentioned in the issue instead of `referenced service port and associate necessary parameters as per need.`

 Also, can you  link the backendConfig spec for easy reference?

Reply:
@kishorv10 Updated documentation.